tests/FunctionsTest.php

```diff
@@ -98,4 +130,118 @@ public function testIterableKeyFirstReturnsNullIfIterableIsGeneratorAndIsEmpty()
             yield;
         })));
     }
+
+    public function testYieldGroupsWithEmptyIterator()
+    {
+        $this->assertEquals([], iterator_to_array(Stdlib\yield_groups(new EmptyIterator(), function () {
+        })));
```

Comment:
```suggestion
        $this->assertEquals([], iterator_to_array(Stdlib\yield_groups(new EmptyIterator(), function () {})));
```

Reply:
I would rather go with `$this->assertEquals([], iterator_to_array(Stdlib\yield_groups(new EmptyIterator(), function () {})));`

Reply:
Ok, that is not allowed 😆

Reply:
Ah. I think you've requested to change it to be a oneliner. Our style checks fail then.